eng/pipelines/templates/stages/cosmos-sdk-client.yml

```diff
@@ -41,7 +41,7 @@ stages:
           vmImage: $(OSVmImage)
 
         steps:
-          - template: ../steps/cosmos-emulator.yml
+          - template: ../../../common/pipelines/templates/steps/cosmos-emulator.yml
```

Comment:
Lets use `/eng/common/pipelines/templates/steps/cosmos-emulator.yml` for the path.

Reply:
Done. Submitted an PR for Java to tweak this as well.

Reply:
Done.